### ISSUE TITLE
icons for gradle and terraform

### ISSUE THREAD

**Issue by MattFenner** [wyze/atom-seti-icons#24]
nice icon set!

would be nice to have icons for gradle, terraform and .bat/.cmd files too

**Comment by MattFenner** [comment]
looks like this is done. awesome.

**Comment by MattFenner** [comment]
actually notice the cmd one is still open

**Comment by wyze** [comment]
Yes, still waiting on .cmd and .bat!

**Comment by GMAGD** [comment]
nice work man! :D

**Comment by wyze** [comment]
`.cmd` and `.bat` were merged in with #53. I'll get a release out shortly.
